client.go

```diff
@@ -127,6 +132,41 @@ func (c *Client) recvVersion() error {
 	return nil
 }
 
+func (c *Client) recv() {
```

Comment:
even though this is a private method, please add a comment as this is an important method